po/sv_SE.po

```diff
@@ -0,0 +1,758 @@
+# Copyright (C) 2020  Matthew "strager" Glazar
+# See end of file for extended copyright information.
```

Comment:
The extended copyright information is missing from the end of the file. Please add it.